src/platforms/apple/common/dsym.mdx

```diff
@@ -46,15 +50,18 @@ An App Store Connect user and password are required, in addition to the normal A
 
 We highly recommend that you create a dedicated App Store Connect user account for the purpose of this integration, and that you **not** use a regular user account.
 
-Using the configured App Store Connect account on a different device will disrupt Sentrys ability to use it in order to download dSYM files.
+Using the configured App Store Connect account on a different device will disrupt Sentry's ability to use it in order to download dSYM files.
 
 </Alert>
 
 These credentials will need to be verified by a two-factor authentication token that you can choose to have sent by SMS.
 
 **Re-authentication**
 
-The App Store Connect credentials and two-factor authentication are short-lived and need to be refreshed on a regular basis. Sentry will prompt users to re-authenticate once the App Store Connect credentials expire.
+The App Store Connect credentials and two-factor authentication are short-lived and need to be refreshed on a regular basis. Sentry will prompt users to re-authenticate in the Project Settings side bar as well as the Custom Repositories section under a project's Debug Files settings once their App Store Connect credentials expire.
```

Comment:
It is also shown in issue details of the project where the source is configured, example:
![image](https://user-images.githubusercontent.com/146720/132010413-bbbea67b-91f6-4869-91db-7e2564456b0d.png)